packages/components/src/__future__/Tabs/subcomponents/Tab/Tab.module.css

```diff
@@ -31,13 +31,18 @@
     opacity: 0.3;
   }
 
+  &[data-disabled]:hover {
+    cursor: default;
```

Comment:
You can actually just chuck `cursor: default` on the `.tab` and you don't need to add this